src/cecs277TermProject/Logger.java

```diff
@@ -0,0 +1,28 @@
+package cecs277TermProject;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+
+/**
+ * Logger is simply our file I/O class, meant to write to the transaction log.
+ */
+public class Logger 
+{
+	
+	private File outFile;
+	private PrintWriter out;
```

Comment:
There should be a fileName variable that store log file name, and expose a getter for that variable so that in main() we can read the log file from the file name in this class.